app/controllers/sessions_controller.rb

```diff
@@ -5,6 +5,11 @@ def new
 
   def create
     session[:auth_user] = auth_hash
+
+    set_connection
+
+    session[:auth_user].user_id = Session.create(first_name: current_user_info[:first_name], last_name: current_user_info[:last_name]).id
```

Comment:
Can we clean this up to be:

```
     session[:auth_user].user_id = Session.create(
       first_name: current_user_info[:first_name],
       last_name: current_user_info[:last_name]
     ).id 
```

Or maybe even use an intermediary variable (because pulling `id` out isn't too obvious).

Reply:
went for variable